custom/succeed/templates/succeed/ucla_table.html

```diff
@@ -1,5 +1,6 @@
-{% extends "reports/async/bootstrap2/tabular.html" %}
+{% extends "reports/async/bootstrap3/tabular.html" %}
 {% load hq_shared_tags %}
+
 {% block js-inline %}
     <script type="text/javascript">
```

Comment:
@kkrampa It looks like `{{block.super}}` is (on purpose?) not being invoked. To fix the loader, I recommend adding the following to `js-inline`
```
    <script type="text/template" id="js-template-loading-report">
        <div class="report-loading-container">
            <div class="report-loading">
                <h4>{% trans "Loading Report" %}</h4>
                <i class="fa fa-spin fa-spinner"></i>
            </div>
        </div>
    </script>
```